src/components/global-toolbar/global-toolbar.js

```diff
@@ -32,7 +38,7 @@ export const GlobalToolbar = ({
             disabled={false}
             icon={LogoIcon}
           />
-          <NavLink exact to={{ pathname: '/' }}>
+          <NavLink exact to={{ pathname: `${sanitizedPathname}` }}>
```

Comment:
```suggestion
          <NavLink exact to={{ pathname: {sanitizedPathname} }}>
```
If I got the syntax right...